frontend/www/js/omegaup/v-tooltip.umd.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module 'v-tooltip' {
+  interface VTooltip {
+    bind: (el: any, _ref?: any) => void;
+    unbind: (el: any) => void;
+    update: (el: any, _ref?: any) => void;
+  }
+  export const VTooltip: VTooltip;
```

Comment:
nit:

```suggestion
  import DirectiveOptions from 'vue';
  export default const VTooltip: DirectiveOptions;
```

así puedes hacer `import { VTooltip } from 'v-tooltip';` y luego referirte a eso como `VTooltip` en vez de `VTooltip.VTooltip`.

Reply:
no había visto lo que sugirió @pabo99 y me gusta más su sugerencia que esta.

Reply:
Cuando conversamos con @pabo99 por Slack, nos percatamos que no existían tipos para esa librería. Por eso terminé haciendo este archivo.

Reply:
ah ya. entonces puedes intentar mi sugerencia (o una variación que funcione)? si podemos evitar el uso de `any`, es mejor.

Reply:
Como estoy haciendo `export default VTooltip`, debo importarlo así: `import VTooltip from 'v-tooltip';`

Sin embargo, me sigue dando un error, que es este: 
```
    ERROR in /opt/omegaup/frontend/www/js/omegaup/components/badge/Badge.vue.ts
    [tsl] ERROR in /opt/omegaup/frontend/www/js/omegaup/components/badge/Badge.vue.ts(40,3)
          TS2345: Argument of type '{ directives: { tooltip: Vue; }; }' is not assignable to parameter of type 'VueClass<Vue>'.
      Object literal may only specify known properties, but 'directives' does not exist in type 'VueClass<Vue>'. Did you mean to write 'directive'?
```

Reply:
Cuando lo invoco así:
```
@Component({
  directives: {
    tooltip: VTooltip.VTooltip,
  },
})
```
El error que recibo es:
```
   ERROR in /opt/omegaup/frontend/www/js/omegaup/components/badge/Badge.vue.ts
    [tsl] ERROR in /opt/omegaup/frontend/www/js/omegaup/components/badge/Badge.vue.ts(41,23)
          TS2339: Property 'VTooltip' does not exist on type 'Vue'.
```
Lo cual tiene sentido.

Reply:
ah entonces hay que hacer

```suggestion
  import { DirectiveOptions } from 'vue'; // faltaban los braces
  const VTooltip: DirectiveOptions;
  export VTooltip;
  export default VTooltip;
```

o un subconjunto de eso. el chiste es que del otro lado uses `import { VTooltip } from 'v-tooltip';` y así no tengas que decir `VTooltip.VTooltip`.

Reply:
Funcionó!!